newIDE/app/src/ObjectsList/index.js

```diff
@@ -429,7 +429,7 @@ const ObjectsList = React.forwardRef<Props, ObjectsListInterface>(
     );
 
     const onAddNewObject = React.useCallback(
-      (item: ObjectFolderOrObjectWithContext) => {
+      (item: ObjectFolderOrObjectWithContext | null) => {
```

Comment:
Is this necessary?

Reply:
Yes, because the selection could be empty and the 1st element of the list is used in some places.
It's to make sure that it's checked at some point.